browser-log.graphqls

```diff
@@ -14,22 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# The list of browser error log
-type BrowserErrorLogBrief {
-    errorLogs: [BasicBrowserErrorLog!]!
-    total: Int!
-}
-
-# Browser error log basic info
-type BasicBrowserErrorLog {
-    uniqueId: String!
-    pagePath: String!
-    time: Long!
-    category: ErrorCategory!
-}
-
 # Represent the conditions used for query BrowserErrorLogBrief
 input BrowserErrorLogQueryCondition {
+    uniqueId: String
```

Comment:
What is log `uniqueId`? How users could get this?

Reply:
from [here](https://github.com/apache/skywalking-data-collect-protocol/blob/b2f381e63702bc43216ef5576637195102302c6e/browser/BrowserPerf.proto#L81), users can copy from the page and share it with others.

Reply:
My point is, this is most likely a UUID only. No one is really caring about it. Also, it could be absent and generated by the OAP automatically.

Reply:
got it, deleted.